### ISSUE TITLE
Zone highlighter/UI

### ISSUE THREAD

**Issue by le-jeu** [wasabee-project/Wasabee-IITC#191]
I could be nice to have a UI to highlight/zoom all links/marker into a zone.

Ideas:
 - Add some column/clickable to the zone dialog
 - grey all stuff not in the selected zone(s?)
 - add a select mode, to point&click link/marker to add/remove to the current selected zone
   - select marker/outgoing links by a rectangle (need a tool to draw the rectangle, cf select region dialog)

**Comment by cloudkucooland** [comment]
Can we assume zones will always be quadrilaterals? If so, we can define them with 4 lat/lng points. Store them in the existing zone struct in the op json (and on server). It seems we could extend quickdraw to "draw zone"

**Comment by cloudkucooland** [comment]
Server support started. Iitc work started on zonepolygons branch

**Comment by cloudkucooland** [comment]
server supports setting a color (distinct from name) and ordered points. After other things settle, I will work on displaying this in a new Leaflet layer.

**Comment by cloudkucooland** [comment]
#259 gets a lot of this.

Can we use zones for selecting regions for multimax and other auto-draws? *ponders*

**Comment by le-jeu** [comment]
That makes sense, but in autodraw you want sometimes discontinued region of portals, or do you want to store a zone as a geojson ?...

**Comment by cloudkucooland** [comment]
I have an idea for selection for auto-draws. Once zonepolygons is merged i will start a new branch.

new uiCommand to create new layer for selection, and draw a polygon on that layer, get portals in selection, delete selection layer… should not be too much work
